### DISCUSSION TITLE
Shorten posts

### DISCUSSION THREAD

**Discussion by hellersqueen** [AprilSylph/XKit-Rewritten#1515] [Help Desk] [answered]
I really like the feature to shorten posts but I was wondering if there was a way to make it work a bit like the collapse queue, as in i'd like to be able to see the footer of the post even if it's shortened ? I can't seem to find a way to make it work.

**Comment by hellersqueen** [comment]
I am **so** sorry for this dumb question. I actually made it work, forget I said anything !

**Reply by marcustyphoon** [reply]
Out of curiosity, was this a question about modifying the extension code for yourself, or just about using it?

**Reply by hellersqueen** [reply, marked as answer]
Modifying I think ? I checked again and I saw no option available to do what I wanted it to so my question was dumb from the start.  
I ended up just using the collapse queue but getting rid of the part that says it's only for drafts and queue so that it would work on my whole dashboard for the time that I need it.  It's not the same thing exactly but it does the job !

**Reply by marcustyphoon** [reply]
Ah gotcha! Yeah, there's definitely a way to mush together the code for those two features to create what you're imagining but it'd be a fair amount of development; I don't think there's a simple tweak you could make to the Shorten Posts code to make it still show the post footer, unfortunately. Tweaking collapse queue definitely works as a quick and dirty solution!
